integrations/docker/images/chip-build-vscode/Dockerfile

```diff
@@ -97,7 +97,7 @@ ENV CY_TOOLS_PATHS="/opt/ModusToolbox/tools_2.4"
 ENV SILABS_BOARD=BRD4161A
 ENV IDF_PATH=/opt/espressif/esp-idf/
 ENV IDF_TOOLS_PATH=/opt/espressif/tools
-ENV IMX_SDK_ROOT=/opt/fsl-imx-xwayland/5.15-kirkstone/
+ENV IMX_SDK_ROOT=/opt/fsl-imx-internal-xwayland/6.1-langdale
```

Comment:
what is the `internal` part? Should this not be the same as the chip-build-imx part?

Also line 49 above seems to have the copy without internal.

Reply:
ah, this is my mistake, I'm apologize... thank you for pointed this!
corrected in the updated PR.